### PR TITLE
[YGert-1] - Add CSVtoSHP and SHP to Network DS func

### PR DESCRIPTION
**Pull request by MokshaSri**
Created CSVtoSHP and testing files
- wrote file to read absolute filepaths from target directory
- Converted CSV to point SHP by first converting episode to geopandas dataframe
- Then geopandas dataframe to point shapefile using ESRI wkt standard 
- More information can be found at https://www.esri.com/arcgis-blog/products/arcgis-pro/mapping/coordinate-systems-difference/

### CODE REVIEW

File: src/convertCSVtoSHP.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+from os.path import join
+
+
+#This function receives a directory and returns an array of all files
+#within the directory, full file paths
+def readDir(inputpath):
+    return [os.path.join(inputpath, file) for file in os.listdir(inputpath)]
+
+
+#The function receives an array of csv epsiode files and
```
Comment (smita7) [inline src/convertCSVtoSHP.py:22]: Please be more specific because its array of file paths

File: src/convertCSVtoSHP.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+#The function receives an array of csv epsiode files and 
+#returns a directory filled with the corresponding point shapefiles 
+def convertCSVtoSHP(filenames):
+    targetPath = os.path.join("../", "SHPfiles/")
+    os.mkdir(targetPath)
+    os.chdir(targetPath)
+    for csvfile in range(len(filenames)):
+        data = pd.read_csv(filenames[csvfile])
+        dataGDF = gpd.GeoDataFrame(data, geometry = gpd.points_from_xy(data['Longitude'], data['Latitude']))
+        ESRI_WKT = 'GEOGCS["GCS_WGS_1984",DATUM["D_WGS_1984",SPHEROID["WGS_1984",6378137,298.257223563]],PRIMEM["Greenwich",0],UNIT["Degree",0.017453292519943295]]'
```
Comment (smita7) [inline src/convertCSVtoSHP.py:31]: Add a comment clarifying what this is

File: src/convertCSVtoSHP.py (hunk 1 above)
Comment (MokshaSri) [reply src/convertCSVtoSHP.py:22]: updated, thanks!

File: src/convertCSVtoSHP.py (hunk 2 above)
Comment (MokshaSri) [reply src/convertCSVtoSHP.py:31]: updated, thanks!

**Review by smita7 — APPROVED** [review]

**Pull request merged**